spi-device.d.ts

```diff
@@ -0,0 +1,46 @@
+export type MODE0 = 0;
+export type MODE1 = 1;
+export type MODE2 = 2;
+export type MODE3 = 3;
+
+export type SpiMode = MODE0 | MODE1 | MODE2 | MODE3;
+
+export type SpiOptions = {
+    mode?: number, // 2-bit, MODE0, MODE1, MODE2, or MODE3, default MODE0
```

Comment:
Should mode be of type SpiMode rather than being of type number?

Reply:
Oh yea, sorry, my bad...
I will fix it soon.